components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-asset.tsx

```diff
@@ -247,12 +243,18 @@ export const PortfolioAsset = (props: Props) => {
       }
       const foundToken = tokensWithCoingeckoId?.find(token => token.coingeckoId.toLowerCase() === coinMarket?.id?.toLowerCase())
       setIsTokenSupported(foundToken !== undefined)
-    } else {
-      setIsTokenSupported(true)
+      return token
     }
-
-    return token
-  }, [assetId, userVisibleTokensInfo, selectedTimeline, tokenId])
+    if (!contractOrSymbol || !chainIdOrSymbol) {
```

Comment:
I think we can remove the `!chainIdOrSymbol` condition since it's already covered at the top of the function.

Reply:
Removed!